src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int SWITCH_BOUND = 5;
+    private static final int RADIUS_OF_DEFAULT_FIGURE = 10;
+    private static final int ORIGIN = 1;
+    private static final int MAX_VALUE = 100;
+    private static final String COLOR_OF_DEFAULT_FIGURE = "white";
+
+    private static final Random random = new Random();
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
Avoid using static for mutable objects. 'colorSupplier' should not be static because it's not a constant and does not need to be shared across instances of FigureSupplier.